src/dodal/devices/p99/sample_stage.py

```diff
@@ -1,18 +1,13 @@
-from ophyd_async.core import Device, SubsetEnum
-from ophyd_async.epics.core import epics_signal_rw
+from ophyd_async.core import StandardReadable, SubsetEnum
+from ophyd_async.epics.core import epics_signal_rw, epics_signal_rw_rbv
 
 
-class SampleAngleStage(Device):
-    def __init__(self, prefix: str, name: str):
-        self.theta = epics_signal_rw(
-            float, prefix + "WRITETHETA:RBV", prefix + "WRITETHETA"
-        )
-        self.roll = epics_signal_rw(
-            float, prefix + "WRITEROLL:RBV", prefix + "WRITEROLL"
-        )
-        self.pitch = epics_signal_rw(
-            float, prefix + "WRITEPITCH:RBV", prefix + "WRITEPITCH"
-        )
+class SampleAngleStage(StandardReadable):
+    def __init__(self, prefix: str, name: str = ""):
+        with self.add_children_as_readables():
+            self.theta = epics_signal_rw_rbv(float, prefix + "WRITETHETA")
+            self.roll = epics_signal_rw_rbv(float, prefix + "WRITEROLL")
+            self.pitch = epics_signal_rw_rbv(float, prefix + "WRITEPITCH")
```

Comment:
The RBV pv is different to the standard so I think we will need to add read_suffix if we are to use epics_signal_rw_rbv
```suggestion
            self.theta = epics_signal_rw_rbv(float, prefix + "WRITETHETA", ":RBV")
            self.roll = epics_signal_rw_rbv(float, prefix + "WRITEROLL", ":RBV"))
            self.pitch = epics_signal_rw_rbv(float, prefix + "WRITEPITCH",":RBV"))
```